app/component/SelectMapLayersDialog.js

```diff
@@ -89,6 +89,61 @@ class SelectMapLayersDialog extends React.Component {
     this.props.setMapMode(mapMode);
   };
 
+  renderMapModeSelector = (config, currentMapMode) => {
+    if (config.CONFIG in GermanConfigs) {
```

Comment:
Can you please replace this check with one that checks if the `bicycle` and satellite keys are present?